src/core/dom/layout/intersection-no-root.js

```diff
@@ -75,10 +75,10 @@ function getInOb(win) {
  * dedupe to the same promise.
  *
  * @param {!Element} el
- * @return {!Promise<IntersectionObserverEntry>}
+ * @return {!Promise<!IntersectionObserverEntry>}
  */
 export function measureIntersectionNoRoot(el) {
-  if (intersectionDeferreds && intersectionDeferreds.has(el)) {
+  if (intersectionDeferreds?.has(el)) {
```

Comment:
Idea for a different PR: Wonder if we should apply a sweep PR for this kind of change.

Reply:
ACK